components/script_traits/lib.rs

```diff
@@ -139,10 +140,10 @@ pub struct LoadData {
     pub data: Option<Vec<u8>>,
     /// The result of evaluating a javascript scheme url.
     pub js_eval_result: Option<JsEvalResult>,
+    /// The referrer.
+    pub referrer: Option<Referrer>,
```

Comment:
Does  this still need to be optional?

Reply:
I'm assuming yes. `None` is passed in 7 places still

Reply:
ok!